assets/src/edit-story/app/story/storyProvider.js

```diff
@@ -125,8 +126,10 @@ function StoryProvider({ storyId, children }) {
       meta: {
         isSaving,
       },
+      reducerState,
     },
     actions: {
+      restore,
```

Comment:
Because `devTools.js:96` will not work, I needed to export the function here, same for state from useStoryReducer (so we don't have to filter the keys from storyProvider state, just export/import the whole reducer state).
Not sure if it's clear but it's all because `devTools.js:96` is not working.

Alternatively can be exported under a different key, because it's only used in devTools right now:
```
state, actions, devTools: { restore, reducerState }
```